schemas/0.11.0/fmu_results.json

```diff
@@ -617,7 +620,7 @@
           "type": "number"
         },
         "xinc": {
-          "exclusiveMinimum": 0,
+          "exclusiveMinimum": 0.0,
```

Comment:
Not sure why this is popping up again.. maybe your `pydantic` version in your venv is not the latest. You could try to pip install pydantic --upgrade and force overwrite the schemas again to see if it disappears 🙂 
Side not it is not picked up as a schema change though by the `update-schemas` script so the overwrite wont happen automatically, so might be that you need to delete the schema upfront first.